### DISCUSSION TITLE
Question 101: Methods and specifics for CE [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#913] [Q&A]
Description: How should I fill in the Method and Specifics for a collection event record with the following description: 

“We used an additional category, ‘Uncertain’ (X), for species that could not easily be assigned to the other categories because they were incidental records obtained outside of our systematic sampling (e.g., during travel between campsites or via indirect evidence). We also included in our list bird species recorded by other researchers during the inventory via photographs, collected feathers or other body parts, accidental mist net captures during bat sampling by Lucía Castro, and observations made while traveling between campsites, at military posts, or in the towns of La Poza and Puerto Galilea. Finally, we obtained Wampis bird names from local scientists to complement the indigenous names reported by Dosantos (2005). Both are presented in Appendix 10 of this book.”

**Comment by rondlg** [comment]
Response:
Methods: Machine observation, human observation
Parse out Methods / Specifics in CE
If useful to tie back to a Survey Protocol Narrative - https://docs.google.com/document/d/1J29pYAzsm7TkdRpIZV7uLLMzCOBJ4AfD_YJsT2d58uI/edit#heading=h.3jz3axbo7dls 
Zach will create record, share irn with Janeen

**Reply by zkachian** [reply]
@fmjjones what would I enter for the record type (ColCollectionType) if it is might be a human observation (direction observation of organism/vocalization, collected feathers, body parts, accidental captures; interviews with local experts) and/or machine observation (photographs) if I can only choose one value for ColCollectionType?

**Comment by fmjjones** [comment]
Use Record Type Sighting.

**Reply by zkachian** [reply]
Thanks, Janeen. When you have a chance, could you please look at CE IRN 3662508 and let me know how the Type, Method, and Specifics look?
